model/generate.go

```diff
@@ -532,7 +533,7 @@ func (g *GeneratedProject) findTasksAndVariantsWithSpecificActivations(requester
 		// Only consider batchtime for mainline builds. We should always respect activate if it is set.
 		if evergreen.ShouldConsiderBatchtime(requester) && bv.hasSpecificActivation() {
 			res.activationVariants = append(res.activationVariants, bv.name())
-		} else if bv.Activate != nil {
+		} else if !utility.FromBoolTPtr(bv.Activate) {
```

Comment:
Nit: we should update that comment on L533 to clarify we only care if activate is false

Reply:
I modified the wording. This comment is a little vague on the difference between activate being explicitly true, explicitly false, and unset, so I tried to clarify it a bit.